smart-contracts/contracts/mixins/MixinTransfer.sol

```diff
@@ -157,26 +157,29 @@ contract MixinTransfer is
    * transfer the key to another account.  This is pro-rated so the fee goes down
    * overtime.
    * @param _owner The owner of the key check the transfer fee for.
+   * @param _time The amount of time to calculate the fee for.
+   * "0" means use all the time remaining on the key for the calculation.
```

Comment:
Might want the comment in the interface to match

Reply:
yes! on it.